src/ci/run.sh

```diff
@@ -64,6 +64,9 @@ RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --enable-locked-deps"
 RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --enable-cargo-native-static"
 RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --set rust.codegen-units-std=1"
 
+# Review: .static-stdcpp, or .static-libstdcpp? Looks like latter.
+RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --set build.static-libstdcpp=1"
```

Comment:
this is under `llvm`, sorry, not `build` - didn't check at first.
```suggestion
RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --set llvm.static-libstdcpp=1"
```

Reply:
Ahhhh. That makes sense, I think.  Trying it... 

Interesting mix of conventions, with some things being `--enable-foo` and others being `foo=1`.  Historical accident depending on who wrote that module, I assume.

Reply:
yeah, it might make sense to clean up at some point, but our CI is a mess in a bunch of other ways too ... no one really maintains these scripts unfortunately.

Reply:
Understood. Low-priority and "not sexy" item compared to other things on the backlog...